browser/rewards_data/rewards_data_sites_util.cc

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/rewards_data/rewards_data_sites_util.h"
+
+#include "base/scoped_observer.h"
+
+namespace {
+
+// This object receives |task_count| calls from RewardsDataRemover, calls
+// |callback| when all tasks are finished and destroys itself.
+class RewardsDataTaskObserver : public content::RewardsDataRemover::Observer {
```

Comment:
what is this for? It doesn't appear to be used anywhere

Reply:
https://github.com/brave/brave-core/pull/1187/files#diff-8581cfbb784a411abd399f14332cbab8R64

Reply:
This file was removed